roles/register_hypervisor/tasks/main.yml

```diff
@@ -28,6 +37,7 @@
                                       --host-name='{{ host_name if host_name is defined else "" }}'
                                       --host-password='{{ root_password }}'
                                       --cluster='{{ cluster_name }}'
+                                      --deploy-hosted-engine
```

Comment:
In 4.1 we no longer able to use the command "hosted-engine --deploy" for additional hosts.
So, to add additional hosts we have to add these hosts (hypervisors) via the sdk API and we have to add this flag "--deploy-hosted-engine" otherwise it will install hypervisor without hosted engine capabilities.